admin-dev/themes/new-theme/scss/components/layout/_nav_bar.scss

```diff
@@ -426,7 +426,8 @@
       }
     }
 
-    @media (max-height: 870px) {			// this two tabs need to be flipped, otherwise
+    @media (max-height: 1040px) {
```

Comment:
This smells bad. :D No idea what magic is this, but targetting a specific pages in global component is not a good idea.

```css
#subtab-ShopParameters.ul-open,
#subtab-AdminAdvancedParameters.ul-open {
```

Also, what is `@media (max-height:` used for? It's very non-standard, I have never ever seen it used in real life. 🤔

Reply:
It's not the page, it targets the elements in the menu that are at the end I totally agree that this is quite dirty, if the structure of the menu is changed is not respected anymore then it will stop working But the purpose is not to refacto this in this PR

The new menu style is a bit wider/higher so the threshold for this rule also needs to be increased basically When the page has enough height there's no need for this because there's enough room to display the submenu

But I agree it would deserve a more generic and stable solution

Reply:
@jolelievre has very well summed up what happen here, and it's clearly quite dirty, but it's not the purpose of the PR, even if we all agree that a better solution would be deserved.